common/src/main/webapp/sass/explore.scss

```diff
@@ -222,13 +224,11 @@ $search-preview-margin: 5px;
   border-radius: 0.28rem;
   justify-content: center;
   align-items: center;
-  background: repeating-linear-gradient(
-    -45deg,
-    black,
-    black 10px,
-    var(--site-border-color) 10px,
-    var(--site-border-color) 11px
-  );
+  background: repeating-linear-gradient(-45deg,
+      black,
```

Comment:
I guess this should be a theme variable?

Reply:
Agreed. This was not introduced in this PR though. It was autoformatted by prettier.